Rubberduck.VBEEditor/SafeComWrappers/VBA/VBProjects.cs

```diff
@@ -173,7 +173,7 @@ private static void OnProjectRenamed(VB.VBProject vbProject, string oldName)
             var projectId = project.ProjectId;
 
             var handler = ProjectRenamed;
-            if (handler != null)
+            if (handler != null && projectId != null)
             {
                 handler(project, new ProjectRenamedEventArgs(projectId, project, oldName));
```

Comment:
@retailcoder - Good eye - probably a copy-paste error.  I'll change it to Invoke when I'm ready to synch again.